example/http/Dockerfile

```
@@ -0,0 +1,24 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang:alpine AS base
+COPY . /go/src/github.com/open-telemetry/opentelemetry-go/
+WORKDIR /go/src/github.com/open-telemetry/opentelemetry-go/
```

So I stole this file and a docker-compose file for my gin-gonic instrumentation example and found some problems. `go install` may work in a strange way when current working directory is one module and we want to install a binary from a submodule. That is, will not work. So I'd recommend the following change:

```suggestion
WORKDIR /go/src/github.com/open-telemetry/opentelemetry-go/example/http/
```